src/pluginList.ts

```diff
@@ -70,5 +70,17 @@ export default [
   {
     name : "web3-plugin-contracts",
     isFeatured : false
-  }
+  },
+  {
+    name : "web3-plugin-sign",
```

Comment:
could you ask https://www.npmjs.com/package/web3-plugin-sign to add github repo in npm package info on NPM